src/components/icons/BoldTextIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Icon from '../../@types/Icon'
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+const BoldTextIcon = ({ style, size }: Icon) => {
```

Comment:
I'd like an icon without a border with a more bold B.

Reply:
I've changed Icon. Right now it's without a border and B is bolder.